packages/astro/src/@types/astro.ts

```diff
@@ -536,6 +536,41 @@ export interface AstroUserConfig {
 	 */
 	redirects?: Record<string, RedirectConfig>;
 
+	/**
+	 * @docs
+	 * @name prefetch
+	 * @type {boolean | PrefetchOptions}
+	 * @description
+	 * Enable prefetching for your site. A prefetch script is automatically added to every page
+	 * in the project. Just add the `data-astro-prefetch` attribute to any `<a />` links on your page
+	 * and you're ready to go!
+	 *
+	 * If you use [View Transitions](https://docs.astro.build/en/guides/view-transitions/), prefetch will
+	 * be enabled automatically with a default of `{ prefetchAll: true }`, which enables prefetching for all links.
+	 * You can configure this option to override the defaults, and even set `false` to disable prefetching
+	 * completely when using View Transitions.
```

Comment:
```suggestion
```
I *think* we can get away without this here. I mentioned up top that it's enabled, and I'll do the same in `prefetchAll`, and we'll mention it both in the guide, and go back and add a note in View Transitions! I think we can keep this config page moving along and still not confuse people!